docs/source/contracts.rst

```diff
@@ -60,3 +63,132 @@ message transfer between L1 and L2.
    ``L1Messenger``             L1
    ``L2Messenger``             L2
    ======================= ===========
+
+
+.. _contracts-parameters:
+
+Parameters
+----------
+
+.. table:: Core contracts parameters
+   :widths: auto
+
+   +----------------------+----------------------------+---------------------------------------+ 
+   | Contract             | Parameter                  | Explanation                           |
+   +======================+============================+=======================================+
+   | RequestManager       | claimStake                 | Minimum amount of source chain's      |
+   |                      |                            | native token to claim.                |
+   |                      +----------------------------+---------------------------------------+
+   |                      | claimRequestExtension      | Additional time given to claim a      |
+   |                      |                            | request.                              |
+   |                      +----------------------------+---------------------------------------+
+   |                      | claimPeriod                | The period for which the claim is     |
+   |                      |                            | valid.                                |
+   |                      +----------------------------+---------------------------------------+
+   |                      | challengePeriodExtension   | Additional time given after challenge |
+   |                      +----------------------------+---------------------------------------+
+   |                      | minFeePPM                  | PPM to determine minLPFee profit for  |
+   |                      |                            | liquidity providers.                  |
+   |                      +----------------------------+---------------------------------------+
+   |                      | lpFeePPM                   | PPM from transfer amount to determine |
+   |                      |                            | LP`s fee.                             |
+   |                      +----------------------------+---------------------------------------+
+   |                      | protocolFeePPM             | PPM from transfer amount to determine |
+   |                      |                            | protocol`s fee.                       |
```

Comment:
We haven't defined `PPM` anywhere, AFAIK. We should mention it somewhere at least once.